### DISCUSSION TITLE
lightdm is pushing dde-session-shell as a dependency in Fedora Linux and not works as it should.

### DISCUSSION THREAD

**Discussion by ilikelinux69** [linuxdeepin/developer-center#6425] [Troubleshooting | 故障排查]
Could you please help to solve the Issue in Fedora Linux and if needed make a separate folder for Fedora as you have for Debian and Archlinux.
We discussed the problem on discourse in the Fedora community:
https://discussion.fedoraproject.org/t/what-happend-to-lightdm-i-tougtht-it-is-light-installed-160mb/98430

If we try to install the lightdm package it will push 160MB because of the dde-session-shell. We do believe it needs some adjustments, and we kindly ask for collaboration that it also works correct on Fedora Linux and an upstream contribution is possible.

Thanks in advance !
Independent Fedorian with alias Ilikelinux :)

**Comment by BLumia** [comment]
> if needed make a separate folder for Fedora as you have for Debian and Archlinux

The existing `debian` and `archlinux` folders are only for CI build purpose, it's not related to redistributing the package to other distro.

I don't use fedora so cannot offer to much help, but if you are facing login-related issue while using lightdm greeter, you can try switch to use SDDM or other greeter instead :)

**Reply by ilikelinux69** [reply]
Ok, the main problem I had was, that when just installing lightdm as a package in Fedora, it is missing a greeter. As deepin is in alpabetically order the first choice it pushes the deep-in related packages where really not are needed.

While choosing the correct greeter package, it will install the necessary dependency, and the installation is much smaller, while it also works.

Sorry when asking here for help and making this confusion. I remember seeing tutorials in internet where you have to install the lightdm package and configuring it while reading the config file. The greeter was always the one from which project it was downloaded.
